src/Components/WithCurrentTime.tsx

```diff
@@ -45,6 +67,6 @@ export class WithCurrentTime extends React.Component<
   }
 
   render() {
-    return this.props.children(this.state.currentTime)
+    return this.props.children(this.state)
```

Comment:
Is there a good reason to push the offset down to children? I think I would prefer it to be added transparently by this component.

Reply:
Yea, the reason for allowing one to opt-in (via `useServerAdjustment`), and then expose it separately, was to keep it opt-in and not add it transparently (and thus affect the other countdown component in Reaction).

I see below you mentioned we should just add that into the other countdown component, so now I'm wondering: should I just remove `useServerAdjustment` and any opt-in, and simply add it transparently? The next time someone needs this _without_ the functionality, they can re-add that sort of a prop back in.

Reply:
Ah, I see- yea that makes better sense! Will do.